src/App/Fossa/FossaAPIV1.hs

```diff
@@ -228,11 +252,10 @@ mkMetadataOpts ProjectMetadata{..} projectName = mconcat $ catMaybes maybes
 
 mangleError :: HttpException -> FossaError
 mangleError err = case err of
-  VanillaHttpException (HTTP.HttpExceptionRequest _ (HTTP.StatusCodeException resp _)) ->
-    case HTTP.responseStatus resp of
-      HTTP.Status 404 _ -> InvalidProjectOrRevision err
```

Comment:
Are we no longer tracking the HTTP status? I feel like the status would be good to at least log if we can.

Reply:
I don't think we were tracking the status previously (ever?). This includes the debug messages, and debug bundle. Different messages were all proxy for some HTTP status.

From support perspective, error UUID, will allow to isolate the request stack trace in backend with respective http status code - we can do this by copy pasting UUID in datadog search.

For end-users, I can see some value having the http status code in error message. I can update error messaging format to reflect this.

Reply:
> Different messages were all proxy for some HTTP status.
I forgot about this behavior, thanks for reminding me. I don't feel strongly about outputting the HTTP status code to the user, with improved API errors I think HTTP errors may just cause confusion.

I do feel strongly that we should include HTTP status codes in the debug messages but I'm not sure this PR is the right place to do that.